com.github.robertsanseries.ciano.json

```diff
@@ -0,0 +1,130 @@
+{
+    "app-id": "com.github.robertsanseries.ciano",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.robertsanseries.ciano",
+    "rename-icon": "com.github.robertsanseries.ciano",
+    "rename-appdata-file": "com.github.robertsanseries.ciano.appdata.xml",
+    "build-options": {
+        "env": {
+            "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR": "/app/share/gir-1.0",
+            "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR": "/app/lib/girepository-1.0"
+        }
+    },
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--device=all",
```

Comment:
same as the previous question. I'm sure you don't need --device=all here, you will likely need --device=dri though to get accelerated codecs.